.codesandbox/ci.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packages": ["packages/core"],
```

Comment:
@devevignesh can you also add rest of the packages

Reply:
sure. I'll add the rest